pkg/auth/oauth/registry/timeoutvalidator.go

```diff
@@ -228,33 +175,21 @@ func (a *TimeoutValidator) flush(flushHorizon time.Time) {
 		flushedTokens, len(tokenList))
 }
 
+func (a *TimeoutValidator) nextTick() time.Time {
+	// Add a small safety Margin so flushes tend to
+	// overlap a little rather than have gaps
+	return time.Now().Add(a.tickerInterval + time.Duration(10)*time.Second)
```

Comment:
I believe `return time.Now().Add(a.tickerInterval + 10*time.Second)` work here so you do not need the type cast.